src/main/java/fr/insee/lunatic/conversion/JSONSerializer.java

```diff
@@ -1,75 +1,29 @@
 package fr.insee.lunatic.conversion;
 
-import fr.insee.lunatic.exception.SerializationException;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
 import fr.insee.lunatic.model.flat.Questionnaire;
-import org.eclipse.persistence.jaxb.MarshallerProperties;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.json.Json;
-import javax.json.JsonObject;
-import javax.json.JsonReader;
-import javax.json.JsonWriter;
-import javax.xml.bind.JAXBContext;
-import javax.xml.bind.JAXBException;
-import javax.xml.bind.Marshaller;
-import java.io.*;
-import java.nio.charset.StandardCharsets;
-
 public class JSONSerializer {
 
 	public JSONSerializer() { }
 
 	private static final Logger logger = LoggerFactory.getLogger(JSONSerializer.class);
 
-	public String serialize(Questionnaire questionnaire) throws JAXBException, UnsupportedEncodingException {
+	public String serialize(Questionnaire questionnaire) throws JsonProcessingException {
```

Comment:
Do we want to expose jackson in the java api of Lunatic-Model?

It's the current standard for this so why not, but we could wrap this jackson exception into on defined in Lunatic-Model (so that client apps don't need to import jackson along with Lunatic-Model for compilation)

Reply:
Yes, i've done this a bit quickly ... wrapping is done bye reverting the previous SerializationException object